internal/cli/actions.go

```diff
@@ -391,16 +391,11 @@ func deleteActionCmd(cli *cli) *cobra.Command {
 				}
 			}
 
-			return ansi.Spinner("Deleting action(s)", func() error {
-				var errs []error
-				for _, id := range ids {
-					if id != "" {
-						if err := cli.api.Action.Delete(cmd.Context(), id); err != nil {
-							errs = append(errs, fmt.Errorf("failed to delete action with ID %q: %w", id, err))
-						}
-					}
+			return ansi.ProgressBar("Deleting action(s)", ids, func(i int, id string) error {
+				if id != "" {
+					return cli.api.Action.Delete(cmd.Context(), id)
```

Comment:
It seems we're now gonna lose the `failed to delete action with ID %q: %w` message.

Reply:
Good catch. I'll make sure it includes that error